theme/templates/resource-landing-page/title-section.html

```diff
@@ -51,6 +51,12 @@
                         <br>
                         <span>Content files are required to publish the resource or make the resource public or discoverable.</span>
                 {% endif %}
+                {%  if cm.resource_type|lower == "collectionresource" %}
+                    {% if not cm.has_resources%}
+                        <br>
+                        <span>Collection should contain at least one resource to be made it published, public or discoverable.</span>
+                    {% endif %}
+                {% endif %}
```

Comment:
@hyi and @zhiyuli The same applies to normal resource. You will get the same message in case of any other resource type when you fill out the required metadata even if there is no content file for that resource. The message is correct but may be confusing. It says the resource has all the metadata to make it public or publish. May be we should change it to say "Metadata Status: Resource now has all the required metadata"